documentation/docs/libraries/overview.md

```diff
@@ -10,32 +10,33 @@ keywords:
 - node.js
 - account
 - multiple
+- explanation 
 ---
 # IOTA Wallet Libraries
 
-The `wallet.rs` library is written in Rust.  You can also find bindings written in Node.js, Python and Java:
+The `wallet.rs` library is primarily written in Rust but you can also find bindings written in Node.js, Python, and Java:
```

Comment:
It is written in Rust only, not primarily. Only its bindings are in different languages (because they act like bridges to the rust library for programs written in other languages).